README.rst

```diff
@@ -0,0 +1,48 @@
+===============
+django-zencoder
+===============
+
+.. image:: https://img.shields.io/pypi/v/django-zencoder.svg
+    :target: https://pypi.python.org/pypi/django-zencoder
+
+.. image:: https://img.shields.io/pypi/pyversions/django-zencoder.svg
+    :target: https://pypi.python.org/pypi/django-zencoder
+
+.. image:: https://img.shields.io/pypi/djversions/django-zencoder
+    :alt: PyPI - Django Version
+    :target: https://pypi.python.org/pypi/django-zencoder
+
+
+Simple integration of a Django model with a FileField with zencoder.com
```

Comment:
Simple integration with zencoder.com

Upload videos and asynchronously store the encoded videos.

Reply:
Klingt besser!